examples/Makefile-cc2538

```diff
@@ -39,7 +39,7 @@ NM                              = arm-none-eabi-nm
 RANLIB                          = arm-none-eabi-ranlib
 OBJCOPY                         = arm-none-eabi-objcopy
 
-BuildJobs                      ?= 99
+BuildJobs                      ?= 10
```

Comment:
I just dialed it down.  99 is perhaps equally arbitrary.  

My VM is stock Ubuntu 16.04 installed with default settings from the iso (1 core, 1GB RAM).  I agree that the reboots I'm seeing are an unexpected result.  It is very easy to replicate, so I can grab any logs that would be helpful.  The issue started somewhat recently; I think the # of files concurrently building in OpenThread may have been less than 99, and now exceeds some threshold of my VM's capabilities.